cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -481,13 +481,19 @@ public OptimizerResult optimizations(ClusterModel clusterModel,
         violatedGoalNamesAfterOptimization.add(goal.name());
       }
 
-      LOG.debug("[{}/{}] Generated {} proposals for {}{}.", optimizedGoals.size(), _goalsByPriority.size(), hasDiff ? "some" : "no",
-                isSelfHealing ? "self-healing " : "", goal.name());
       step.done();
       if (LOG.isDebugEnabled()) {
         LOG.debug("Broker level stats after optimization: {}", clusterModel.brokerStats(null));
```

Comment:
Is it necessary to also add this to Info level

Reply:
Good question!

I believe this is going to be very helpful for identifying production issues.

Also this will not make the logs look verbose, because the code path is only hit on start of each rebalance. It will provide critical meta-level information of the goal optimzation.